lib/assets/javascripts/cartodb/table/mapview.js

```diff
@@ -559,7 +584,8 @@ cdb.admin.MapTab = cdb.core.View.extend({
     this.mapView = new mapViewClass({
       el: div,
       map: this.map,
-      user: this.user
+      user: this.user,
+      readOnly: this.vis.get('type') !== 'derived'
```

Comment:
Ah, I see, `readOnly` might be a bit misnamed? You could just do the type === 'derived' check inside the map view. It should have the vis object.